include/cuco/detail/open_addressing_impl.cuh

```diff
@@ -135,6 +137,55 @@ class open_addressing_impl {
     this->clear_async(stream);
   }
 
+  /**
+   * @brief Constructs a statically-sized open addressing data structure with the number of elements
+   * to insert, sentinel values, the desired load factor and CUDA stream
```

Comment:
I think this warrants more explanation about what `n` means, especially how this construct differs from the existing one that just takes a capacity.

Reply:
@jrhemstad is it better now?